scripts/main.js

```diff
@@ -54,11 +54,15 @@
             return;
         }
 
+        // Crown submission results in either the boolean `false`, or the total submitted crowns.
         if (ev.data.jacks_message === 'crownSubmissionStatus') {
-            if (ev.data.submitted) {
-                displayFlashMessage(ev.data.settings, "success", "Thank you for submitting crowns!");
+            const counts = ev.data.submitted;
+            if (counts) {
+                displayFlashMessage(ev.data.settings, "success",
+                    `Submitted ${counts} crowns for `
+                    +  $('span.hunterInfoView-userName').text() + ".");
```

Comment:
Couldn't this be consolidated into the template literal?

```js
`Submitted ${counts} crowns for ${$('span.hunterInfoView-userName').text()}.`
```

Reply:
Yep, it could. I originally was outputting more info (G/S/B individual) and broke it across lines to limit editor scrolling. Since the success flash is only shown for 1.5s it doesn't make sense to have a lot of detailed info in it.